src/main/java/uk/gov/pay/products/resources/ProductResource.java

```diff
@@ -155,4 +156,13 @@ public Response findProductByProductPath(
                 .map(product -> Response.status(OK).entity(product).build())
                 .orElseGet(() -> Response.status(NOT_FOUND).build());
     }
+
+    @GET
+    @Path("/stats/products")
+    @Produces(APPLICATION_JSON)
+    public Response findProductsAndStats(@QueryParam("gatewayAccountId") Integer gatewayAccountId) {
+        logger.info("Listing all live products and usage stats");
```

Comment:
I'll add gateway account id to the logs, great suggestion 👍 this will also show sandbox account products but only ones that are 'demos' or 'prototypes' -- I'll update the wording on this log line